### PR TITLE
Charts - Move Font type to common/charts

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

File: packages/devextreme/js/common/charts.d.ts [hunk 1]
```diff
@@ -341,6 +338,43 @@ export type GradientColor = {
     color: string;
 };
 
+/**
+ * @docid
+ * @namespace DevExpress.common.charts
+ * @public
+ */
+export type Font = {
```
Comment (mpreyskurantov) [inline packages/devextreme/js/common/charts.d.ts:346]: "devextreme-internal-tools" fix is required

**Review by mpreyskurantov — CHANGES_REQUESTED** [review]

File: packages/devextreme/js/viz/core/base_widget.d.ts [hunk 2]
```diff
@@ -915,3 +883,13 @@ export interface BaseWidgetAnnotationConfig {
      */
     y?: number;
 }
+
+// #region deprecated in 22.2
```
Comment (mpreyskurantov) [inline packages/devextreme/js/viz/core/base_widget.d.ts:887]: 22.2 or 23.2?

File: packages/devextreme/js/viz/core/base_widget.d.ts [hunk 3]
```diff
@@ -915,3 +883,13 @@ export interface BaseWidgetAnnotationConfig {
...
+
+/**
+ * @namespace DevExpress.viz
+ * @deprecated Use Font from DevExpress.common.charts.Font instead
```
Comment (IlyaKhD) [inline packages/devextreme/js/viz/core/base_widget.d.ts]:
```suggestion
 * @deprecated Use Font from common/charts instead

```

File: packages/devextreme/js/viz/core/base_widget.d.ts [hunk 4]
```diff
@@ -32,6 +32,7 @@ import {
     TextOverflow,
     Theme,
     WordWrap,
+    Font as CommonFont,
```
Comment (IlyaKhD) [inline packages/devextreme/js/viz/core/base_widget.d.ts:35]: We can avoid using renaming here:
```suggestion
    Font,
```
I've tried it locally - seems to work fine:


```ts
/**
 * @namespace DevExpress.viz
 * @deprecated Use Font from DevExpress.common.charts.Font instead
 */
export type Font = Font;
```

Comment (IlyaKhD) [reply packages/devextreme/js/viz/core/base_widget.d.ts:35]: And you'll have to add a line to the aliases.d.ts file with different deprecation message

Comment (IlyaKhD) [reply packages/devextreme/js/viz/core/base_widget.d.ts:35]: Nope, this doesn't work. Behaves as expected, there is no magic in this world(

File: packages/devextreme/js/common/charts.d.ts (hunk 1 above)
Comment (alexanderPolosatov) [reply packages/devextreme/js/common/charts.d.ts:346]: fixed

**Review by mpreyskurantov — APPROVED** [review]

File: packages/devextreme/js/viz/core/base_widget.d.ts (hunk 3 above)
Comment (alexanderPolosatov) [reply packages/devextreme/js/viz/core/base_widget.d.ts]: fixed

**Review by marker-dao — APPROVED** [review]

**Pull request merged**